src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,7 @@ Pending
 18.0.0b18
 +++++++
 * Add validation error when neither --location or --cluster and --resource-group-name are specified for az extension type list or az extension type version list
+* Add support for `ManagedSystem` Agent Pool Mode.
```

Comment:
please move it to below the `Pending` part
```suggestion
* Add support for `ManagedSystem` Agent Pool Mode.

18.0.0b18
+++++++
* Add validation error when neither --location or --cluster and --resource-group-name are specified for az extension type list or az extension type version list
```

Reply:
As yan suggested, please move your release note to the pending section if you don't want to release a new version immediately. Otherwise, please add a new section with version number 18.0.0b19 and update setup.py accordingly.

Reply:
sure, updated.